app/templates/_package.json

```diff
@@ -35,7 +35,11 @@
     "<%= props.jsPreprocessor.module %>": "<%= props.jsPreprocessor.version %>",
     "gulp-sourcemaps": "~1.3.0",
 <% } if (props.jsPreprocessor.srcExtension === 'es6') { %>
-    "gulp-browserify": "~0.5.0",
+    "browserify": "~8.1.1",
```

Comment:
I think it's exactly why we will probably never switch to some `gulp-tsd` plugin :)